### DISCUSSION TITLE
Getting weekly box score

### DISCUSSION THREAD

**Discussion by Shivang-P** [cwendt94/espn-api#184] [ESPN API]
Hey, just wanted to say this library has been amazing so far! I am trying to build out a data visualization project and wanted to get past game logs for specific players but not sure how to go about this.

When I try something like the following to get a past game log, 

id = league.teams[0].roster[0].playerId
player = league.player_info(playerId=id)

I get a AttributeError: 'League' object has no attribute 'player_info' error.

**Comment by cwendt94** [comment]
Thanks @Shivang-P! Is this for Football or Basketball API? The Football League should have `player_info` function. 

What version of the package are you running? You can run `pip show espn_api` for the package version!
